library/common/extensions/filters/http/platform_bridge/filter.h

```diff
@@ -77,45 +79,54 @@ class PlatformBridgeFilter final : public Http::PassThroughFilter,
   Http::FilterTrailersStatus encodeTrailers(Http::ResponseTrailerMap& trailers) override;
 
 private:
-  static void replaceHeaders(Http::HeaderMap& headers, envoy_headers c_headers);
-
-  Http::FilterHeadersStatus onHeaders(Http::HeaderMap& headers, bool end_stream,
-                                      envoy_filter_on_headers_f on_headers);
-
-  Http::FilterDataStatus onData(Buffer::Instance& data, bool end_stream,
-                                Buffer::Instance* internal_buffer,
-                                Http::HeaderMap** pending_headers, envoy_filter_on_data_f on_data);
-
-  Http::FilterTrailersStatus onTrailers(Http::HeaderMap& trailers,
-                                        Buffer::Instance* internal_buffer,
-                                        Http::HeaderMap** pending_headers,
-                                        envoy_filter_on_trailers_f on_trailers);
-
-  // Scheduled on the dispatcher when resumeRequest is called from platform
-  // filter callbacks. Provides a snapshot of pending request state to the
-  // platform filter, and consumes invocation results to modify pending HTTP
-  // entities before resuming decoding.
-  void onResumeDecoding();
-
-  // Scheduled on the dispatcher when resumeResponse is called from platform
-  // filter callbacks. Provides a snapshot of pending response state to the
-  // platform filter, and consumes invocation results to modify pending HTTP
-  // entities before resuming encoding.
-  void onResumeEncoding();
+  /**
+   * Internal delegate for managing logic and state that exists for both the request (decoding)
+   * and response (decoding) paths.
```

Comment:
shouldn't one of these be `encoding` rather than `decoding`?

Reply:
Yep, good catch.